frontend/public/components/utils/status-box.tsx

```diff
@@ -60,7 +60,9 @@ LoadingBox.displayName = 'LoadingBox';
 
 export const EmptyBox: React.FC<EmptyBoxProps> = ({ label }) => (
   <Box>
-    <div className="text-center">{label ? `No ${label} Found` : 'Not Found'}</div>
+    <div data-test-id="empty-message" className="text-center">
```

Comment:
assuming this is an unrelated format fix.

Reply:
Yes, but also added `data-test-id`

Reply:
data-cy?